resources/views/dashboard/settings/app-setup.blade.php

```diff
@@ -134,6 +134,17 @@
                                 </div>
                             </div>
                         </div>
+                        <div class="row">
+                            <div class="col-xs-12">
+                                <div class="checkbox">
+                                    <label>
+                                        <input type="hidden" value="0" name="app_only_display_incidents">
+                                        <input type="checkbox" value="1" name="app_only_display_incidents" {{ Config::get('setting.app_only_display_incidents') ? 'checked' : null }}>
```

Comment:
We don't use the `Config` facade here, instead we pass through a value, see how the rest are done 👍

Reply:
Done 👍